.github/ISSUE_TEMPLATE/enhancement.yml

```diff
@@ -29,3 +29,15 @@ body:
       description: Please describe the potential enhancement and its use cases.
     validations:
       required: true
+
+  - type: dropdown
+    id: targetVersion
+    attributes:
+      label: Gluten version
+      description: Please provide the target gluten version you want to support.
+      options:
+        - Gluten-1.5.x
+        - Gluten-1.6.x
+        - N/A
```

Comment:
Suggestion: Remove `-N/A` and replace it by `- main branch` and place it at the top.